packages/styles/stories/Components/menu/menu-icon.example.html

```diff
@@ -15,14 +16,76 @@
+                <span class="fd-menu__title">Option 4</span>
+                <span class="fd-menu__shortcut">Ctrl + A</span>
+            </a>
+        </li>
+        <li class="fd-menu__item" role="presentation">
+                <a class="fd-menu__link" href="#" role="menuitem">
+                    <span class="fd-menu__addon-before"><i class="sap-icon--cart" role="presentation"></i></span>
+                    <span class="fd-menu__title">Option 4</span>
+                    <span class="fd-menu__shortcut">Ctrl + A</span>
+                    <span class="fd-menu__addon-after"><i class="sap-icon--history" role="presentation"></i></span>
+                </a>
+        </li>
+    </ul>
+</nav>
+
+<br><br>
+<h3>Compact</h3>
+<nav class="fd-menu fd-menu--icons fd-menu--icons--compact">
+    <ul class="fd-menu__list" role="menu">
+        <li class="fd-menu__item" role="presentation">
+                <a class="fd-menu__link" href="#" role="menuitem">
+                    <span class="fd-menu__addon-before"><i class="sap-icon--grid" role="presentation"></i></span>
+                    <span class="fd-menu__title">Option 1</span>
+                    <span class="fd-menu__addon-after"><i class="sap-icon--wrench" role="presentation"></i></span>
+                </a>
+        </li>
         <li class="fd-menu__item" role="presentation">
                 <a class="fd-menu__link" href="#" role="menuitem">
+                    <span class="fd-menu__addon-before"><i class="sap-icon--accept" role="presentation"></i></span>
+                    <span class="fd-menu__title">Option 2</span>
+                    <span class="fd-menu__addon-after"><i class="sap-icon--history" role="presentation"></i></span>
+                </a>
+        </li>
+        <li class="fd-menu__item" role="presentation">
+                <a class="fd-menu__link is-focus" href="#" role="menuitem">
+                    <span class="fd-menu__addon-before"><i class="sap-icon--wrench" role="presentation"></i></span>
+                    <span class="fd-menu__title">Option 3</span>
+                    <span class="fd-menu__addon-after"><i class="sap-icon--lightbulb" role="presentation"></i></span>
+                </a>
+        </li>
+        <li class="fd-menu__item" role="presentation">
+            <a class="fd-menu__link" href="#" role="menuitem">
+                <span class="fd-menu__title">Option 3</span>
+                <span class="fd-menu__addon-after"><i class="sap-icon--lightbulb" role="presentation"></i></span>
+            </a>
+        </li>
+        <li class="fd-menu__item" role="presentation">
+            <a class="fd-menu__link" href="#" role="menuitem">
+                <span class="fd-menu__addon-before"><i class="sap-icon--cart" role="presentation"></i></span>
+                <span class="fd-menu__title">Option 4</span>
+                <span class="fd-menu__shortcut">Ctrl + A</span>
+            </a>
```

Comment:
Can we not add new Compact examples? I'm trying to remove them from everywhere else 😆